pkg/metrics/subsystems.go

```diff
@@ -5,11 +5,10 @@ type Subsystem string
 
 // These consts enumerate all the subsystems that expose Prometheus metrics.
 const (
-	CentralSubsystem       Subsystem = "central"
-	SensorSubsystem        Subsystem = "sensor"
-	ComplianceSubsystem    Subsystem = "compliance"
-	NodeInventorySubsystem Subsystem = "nodeinventory"
```

Comment:
This is not required anymore as Node Inventory container does not produce metrics on its own.

Reply:
I added that in the previous PR (#7224) and this was never used (even in dev), so no breaking change :) Thanks for checking the details!